### PR TITLE
Add package soundex

### PR DESCRIPTION
**Pull request by Kashiwara0205**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -18504,5 +18504,17 @@
     "description": "Nim CMAES library",
     "license": "Apache-2.0",
     "web": "https://github.com/zevv/nimcmaes"
+  },
+  {
+    "name": "soundex",
+    "url": "https:///",
```
Comment (Yardanico) [inline packages.json:18510]: That doesn't look correct for me :P

Comment (Kashiwara0205) [reply packages.json:18510]: Thank you for the review.
I fix it and submit again.

Comment (Kashiwara0205) [reply packages.json:18510]: I checked about URL setting but I don't know how to set the URL.
How should I set the URL.

If it's not too much trouble, can you tell me?

**Comment by Kashiwara0205** [comment]
Ｉfix package.json

**Pull request merged**